integration/setup/setup.go

```diff
@@ -163,6 +163,8 @@ func SetupWithOpts(tb testtb.TB, opts *SetupOpts) *SetupResult {
 
 	collection := setupCollection(tb, setupCtx, client, opts)
 
+	setupUser(tb, setupCtx, client)
```

Comment:
btw, does it make sense to call `dropUser` before we set it up? so, if the user already exists (e.g. from the previous test run), `createUser` doesn't return an error.

Reply:
No. Please explain why :)

Reply:
Ah, nevermind, In my case, I do it for MongoDB, not for FerretDB (and it's MongoDB that returns an error, not FerretDB).  And in this setup it's not created for MongoDB anyway.

Reply:
There should be a cleanup in `tb.Cleanup` being called - `dropAllUsersFromDatabase` is executed. What I don't understand is why it doesn't work for MongoDB. But probably I created it with some different params.